phpunit/plugins/time-to-read-block/renderBlock.php

```diff
@@ -0,0 +1,184 @@
+<?php
```

Comment:
Interesting choice, should it be `phpunit/plugins/time-to-read-block` or `plugins/time-to-read-block/phpunit`?

Reply:
I guess, similar question for e2e tests.

Reply:
> Interesting choice, should it be phpunit/plugins/time-to-read-block or plugins/time-to-read-block/phpunit?

That seems reasonable and makes the tests easier to find when they're colocated with the plugins.

That said, I think it's important to be able to run all tests of a particular type using the same command, from the root of the repo, rather than needing to know about separate commands or switch directories to run a complete set of tests. Additionally I think we should avoid duplicating testing configuration as much as possible.

It would also be good to define a separate plugins suite for phpunit, and something similar for other test types, so it's trivial to run plugin tests separately, when desired.

Reply:
> Additionally I think we should avoid duplicating testing configuration as much as possible.

💯 

In my mind the root level is basically `plugins/gutenberg` but for historic reasons it's in the root. In other words, I like when tests are collocated with the plugin I think, not sure how complex would it be to do that.

 - Leave the configuration in a root folder.
 - Collocate the plugin tests.
 - For Gutenberg it's a special case, maybe keep it root level for now.